app/components/Settings/Preview/TableTree.react.js

```diff
@@ -27,6 +22,31 @@ class TableTree extends Component {
         })
     }
 
+    /**
+     *
+     * @param {object} props - Component Props
+     * @param {object} props.schemaRequest - Schema Request
+     * @param {function} props.getSqlSchema - Get SQL Schema Function
+     * @param {function} props.updatePreview - Update Preview Function
+     * @param {object} props.preview - Preview details
+     * @param {object} props.connectionObject - Connection Object
+     * @param {object} props.connectionObject.database - Database Name
+     * @param {object} props.connectionObject.dialect - Dialect for DB
+     * @param {object} props.connectionObject.storage - Storage
+     */
+    constructor(props) {
```

Comment:
Since you're working on this file. Please, could remove this constructor? The default constructor is enough for this component.